includes/admin/forms/class-metabox-form-data.php

```diff
@@ -87,12 +87,12 @@ function setup() {
 
 
 	/**
-	 * Get metabox settings
-	 *
-	 * @since 1.8
-	 *
-	 * @return array
-	 */
+     * Get metabox settings
+     *
+     * @since      1.8
+     *
+     * @return array
+     */
 	function get_settings() {
```

Comment:
Are there any meaningful changes to this file - or is it just formatting?

Reply:
I initially had changes in here and then went a different route. It's just formatted now.